schema-statistics/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Tracking Postgres schema statistics in pganalyze'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+As part of the "Schema Statistics" functionality, pganalyze allows tracking workload patterns, schema and data model design, as well as changes over time.
+
+## Breakdown of table and index sizes
+
+![](table-treemap.png)
+
+In the schema statistics overview you can find two treemaps, as well as full lists of all the tables and indexes on a given database. The treemap is useful to identify how a specific index or table size relates to the total.
+
+## Table growth over time
+
+![](table-growth.png)
+
+For each table you can track growth over time, understand how a table utilizes TOAST (The Oversized Attribute Storage Technique) for storing large values, and see the aggregate size for all indexes on the table.
```

Comment:
Well now I'm getting less confident about leaving this review but I still think having the comma reads better. Ditto to below two paragraphs.

```suggestion
For each table, you can track growth over time, understand how a table utilizes TOAST (The Oversized Attribute Storage Technique) for storing large values, and see the aggregate size for all indexes on the table.
```